src/templates/app/user/metatrader.jsx

```diff
@@ -190,12 +190,12 @@ const Metatrader = () => (
                         </div>
                         <div className='mt-link-web'>
                             <div className='mt5-web-link'>
-                                <a href='https://trade.mql5.com/trade?servers=Deriv-Demo&trade_server=Deriv-Demo' target='_blank' rel='noopener noreferrer'>
+                                <a id='mt5_web_link_demo' href={`${it.url_for('user/metatrader/metatrader_terminal')}?trade_server=Deriv-Demo&servers=Deriv-Demo`}>
                                     {it.L('Demo')}
                                 </a>
                             </div>
                             <div className='mt5-web-link'>
-                                <a href='https://trade.mql5.com/trade?servers=Deriv-Server&trade_server=Deriv-Server' target='_blank' rel='noopener noreferrer'>
+                                <a id='mt5_web_link_real' href={`${it.url_for('user/metatrader/metatrader_terminal')}?trade_server=Deriv-Server&servers=Deriv-Server`}>
```

Comment:
do we need these ids?

Reply:
Nope, I added them as I initially wanted to assign the `href` URL programmatically, but decided to do it directly with `string` template instead since we're not going to dynamically need to update it on the dashboard.